app/controllers/book_suggestions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BookSuggestionsController < ApiController
```

Comment:
Since you aren't doing an API here, extend from ApplicationController instead of ApiController